src/Sylius/Bundle/ApiBundle/Serializer/Denormalizer/ProductAttributeValueDenormalizer.php

```diff
@@ -67,6 +68,10 @@ public function getSupportedTypes(?string $format): array
      */
     private function denormalizeValue(array $data): array
```

Comment:
And then no move checks or function calls needed.
```suggestion
    private function denormalizeValue(array $data, ProductAttributeInterface $attribute): array
```